Source/WebCore/platform/graphics/Path.cpp

```diff
@@ -460,13 +461,8 @@ static FloatRect computeArcBounds(const FloatPoint& center, float radius, float
         return { center.x() - radius, center.y() - radius, diameter, diameter };
     }
 
-    auto normalize = [&] (float radians) {
-        double circles = radians / fullCircle;
-        return fullCircle * (circles - floor(circles));
-    };
-
-    start = normalize(start);
-    end = normalize(end);
+    start = normalizeAngle(start);
+    end = normalizeAngle(end);
```

Comment:
`fullCircle` is still defined and used above. Can we use radiansPerTurnFloat instead?

Reply:
ops, thanks! Replacing by `radiansPerTurnFloat` that is used now, solving the other comment.